### PR TITLE
Add `winkern` (ttf_flag_oldkernmappedonly) for Python

### PR DESCRIPTION
**Pull request by ctrlcctrlv**
Previously, it was only possible to add Windows-compatible kern in the
GUI. This makes it so you can pass a `winkern` flag to enable this
function in scripts, which is the last thing keeping me from totally
automating my build.

Provide a general summary of your changes in the **Title** above.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### Motivation and Context
- [x] Why is this change required? What problem does it solve?
It allows me to automate my build rather than use the GUI.

- [x] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.
N/A

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [x] Describe your changes in detail.
See above.

### Final checklist
Go over all the following points and check all the boxes that apply. 
If you're unsure about any of these, don't hesitate to ask. We're here to help! Various areas of the codebase have been worked on by different people in recent years, so if you are unfamiliar with the general area you're working in, please feel free to chat with people who have experience in that area. See the list [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#people-to-ask).
- [x] My code follows the code style of this project found [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#coding-style).
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md) guidelines.

Your pull request will be tested via Travis CI to automatically indicate that your changes do not prevent compilation. FontForge is a big program, so Travis can easily take over 20 minutes to confirm your changes are buildable. Please be patient. More details about using Travis can be found [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#using-travis-ci).  
  
If it reports back that there are problems, you can log into the Travis system and check the log report for your pull request to see what the problem was. If no error is shown, just re-run the Travis test for your pull-request (that failed) to see a fresh report since the last report may be for someone else that did a later pull request, or for mainline code. If you add new code to fix your issue/problem, then take note that you need to check the next pull request in the Travis system. Travis issue numbers are different from GitHub issue numbers.

### CODE REVIEW

**Comment by ctrlcctrlv** [comment]
@skef I noticed that too, I even tried `0x20000000` at first but that did not work. The change I've submitted however does work, I tested it by generating a font with the GUI and then with a script and diffing them. Furthermore, the script generated the Windows-only kern warning (https://github.com/fontforge/fontforge/blob/master/fontforge/tottfaat.c#L286) when I used `0x20000000`, but does not with `0x4000000`.

**Comment by skef** [comment]
What prompted you to try 0x4000000 in the first place?

**Comment by skef** [comment]
Ah, never mind -- the magic key is savefont.c:1198 . I'll try to figure out why there are two mappings ...

**Comment by ctrlcctrlv** [comment]
This https://github.com/fontforge/fontforge/blob/master/fontforge/savefont.c#L1241

**Comment by ctrlcctrlv** [comment]
Yes, you found it :)

**Comment by skef** [comment]
Ugh. So basically savefont.c:GenerateScript has a private flagset and the original scripting language just has users pass in an int. The documentation being [here](https://fontforge.github.io/scripting-alpha.html#G). When python support was added in included a textual flag interface, but the code was never adjusted to put the mapping in a common place. 

@ctrlcctrlv, you're a solid upstanding citizen, so how about this proposal:

1. Look at `ttf_flags` in `fontforge/splinefont.h`
2. Using that model, add an `fm_flags` to `savefont.h` with all the values from `savefont.c:GenerateScript`.
3. Modify `GenerateScript` to use those names rather than the numbers
4. Do the same in `python.c:gen_flags`. 

I know this should not be "your problem", but it would definitely improve the code base. I can advise (or finish up if that makes more sense).

**Comment by ctrlcctrlv** [comment]
Thank you for calling me a "solid upstanding citizen", but my C-fu is definitely not up to snuff to do what you're asking me to do, I'm sorry. It took me quite a while to get this working how it is, I doubt I can successfully do a refactor.

**Comment by ctrlcctrlv** [comment]
Personally I think you should commit this and do the refactor as another PR

**Comment by skef** [comment]
Who says I have merge authority? 

But never mind -- I'm all up in your repo.

**Comment by ctrlcctrlv** [comment]
Yeah I'm sorry @skef. Thanks for your help. I am working on learning C better so I can help more with FontForge. :)

**Comment by jtanx** [comment]
@skef feel free to merge stuff if it looks good to you

File: fontforge/savefont.h [hunk 1]
```diff
@@ -32,6 +32,35 @@
 
 #include "splinefont.h"
 
+enum fm_flags { fm_flag_afm = 0x1,
```
Comment (jtanx) [inline fontforge/savefont.h]: This is an odd way of indenting...

**Pull request merged**